CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+cmake_minimum_required(VERSION 3.3)
+project(EDM4HEP VERSION 0.1.0)
+
+#--- Run the podio class generator and link library -------
+
+find_package(podio REQUIRED HINTS $ENV{PODIO})
+
+execute_process(COMMAND python ${podio_CMAKE_DIR}/../python/podio_class_generator.py edm4hep.yaml edm4hep edm4hep
+                WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+find_library(PODIOLIB podio PATHS ${podio_DIR}/../lib)
+if (NOT PODIOLIB)
+    message(FATAL_ERROR "libpodio.so(dylib) cannot be found dynamically. Make sure you have set up your environment to use PODIO")
+endif()
+link_directories(${podio_LIBRARY_DIR})
+
+#--- Declare ROOT dependency ---------------------------------------------------
+list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS})
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+find_package(ROOT REQUIRED COMPONENTS RIO Tree Physics)
+include_directories(${ROOT_INCLUDE_DIR})
+include(${ROOT_USE_FILE})
+link_directories(${ROOT_LIBRARY_DIR})
+set(ROOT_genreflex_cmd ${ROOT_genreflex_CMD})
+
+# set up include-directories
+include_directories(
+  ${PROJECT_SOURCE_DIR}
+  ${PROJECT_BINARY_DIR}
+  ${ROOT_INCLUDE_DIR}
+  ${podio_INCLUDE_DIRS}
+  )
+
+#--- Define basic build settings -----------------------------------------------
+# - Use GNU-style hierarchy for installing build products
+include(GNUInstallDirs)
+
+# Offer the user the choice of overriding the installation directories
+set(INSTALL_LIB_DIR lib CACHE PATH "Installation directory for libraries")
+set(INSTALL_BIN_DIR bin CACHE PATH "Installation directory for executables")
```

Comment:
Both shouldn't be needed as GNUInstallDirs *should* be providing lib/bin install locations.